### PR TITLE
Class 10

### PR DESCRIPTION
**Pull request by jimflores5**

### CODE REVIEW

File: Class-10-(Thymeleaf-views).md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Class 10: Thymeleaf and Views
+
+## In the prep work this week, the students learned
```
Comment (gildedgardenia) [inline Class-10-(Thymeleaf-views).md:3]: They also learned how to link CSS files/ other static resources and add photos and things!

File: Class-10-(Thymeleaf-views).md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+1. Explain why ``th:unless`` it not quite analogous to the ``else`` clause in a conditional block.
+
+    1. ``th:if`` can be made to match ``th:unless`` by adding a ``not`` operator (e.g. ``th:unless=${num == 5}`` and ``th:if="${num != 5}"`` accomplish the same thing). Discuss whether using one format or the other is a personal preference or a common convention.
+
+1. Show examples of how to use ``th:fragment`` and ``th:replace``.
+
+## Small Group Time: Class 10 Studio (TA Notes)
+
+1. Do your best to help students who may be stuck with IntelliJ errors or warnings related to Gradle or JDK versions.
+1. The students will not have to create a new project from scratch, so nudge them away from [start.spring.io](start.spring.io) if necessary.
```
Comment (gildedgardenia) [inline Class-10-(Thymeleaf-views).md]: I would rephrase this. If they read the directions, they would know this. Maybe, "Walk around and make sure no one is struggling with forking, cloning, and opening starter code in IntelliJ"?

**Review by gildedgardenia — APPROVED** [review]
Looks great! Only 2 small comments!

File: Class-10-(Thymeleaf-views).md (hunk 1 above)
Comment (jimflores5) [reply Class-10-(Thymeleaf-views).md:3]: Added.  Thanks.

File: Class-10-(Thymeleaf-views).md (hunk 2 above)
Comment (jimflores5) [reply Class-10-(Thymeleaf-views).md]: Fixed.  Thanks.
